torchgeo/datasets/skippd.py

```diff
@@ -140,12 +162,18 @@ def _load_image(self, index: int) -> Tensor:
         import h5py
 
         with h5py.File(
-            os.path.join(self.root, self.data_dir, self.img_file_name), "r"
+            os.path.join(self.root, self.data_file_name.format(self.task)), "r"
         ) as f:
-            arr = f[self.split]["images_log"][index, :, :, :]
+            arr = f[self.split]["images_log"][index]
+
+        # forecast has dimension [16, 64, 64, 3] but reshape to [64, 64, 48]
```

Comment:
I'm guessing this should be:
```suggestion
        # forecast has dimension [16, 64, 64, 3] but reshape to [48, 64, 64]
```